styles.css

```diff
@@ -0,0 +1,79 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+}
+
+.text {
+  width: 80%;
+  margin: 0 auto;
+  margin-top: 5em;
+}
+
+h1 {
+  font-size: 40px;
+  font-weight: 700;
+  width: 60%;
+  color: #172b4d;
+  line-height: 52px;
+}
+
+.paragraph1 {
+  margin: 15px 0;
+  width: 80%;
+  margin-left: 0;
+}
+
+.lets-connect {
+  font-size: 16px;
+  font-weight: 500;
+  color: #7f8cff;
+  margin-left: 0%;
+}
+
+.buttons li {
+  display: inline-block;
+  margin-right: 20px;
+  margin-top: 25px;
+  margin-left: 0%;
+}
+
+.buttons li a {
+  cursor: pointer;
+}
+
+.menu-icon {
+  cursor: pointer;
+}
+
+.nav-bar {
+  width: 80%;
+  height: 70px;
+  color: #565b86;
+  display: flex;
+  justify-content: space-between;
+  margin: 0 auto;
+  align-items: center;
+}
+
+.logo {
+  cursor: pointer;
+  font-size: 18px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 20px;
+  color: #6070ff;
+}
+
+#section-one {
+  background-image: url('./images/background.png');
+  background-size: cover;
+  background-repeat: no-repeat;
+  width: 100%;
+}
```

Comment:
- [ ] Let us change ids with classes, kindly, change your id with class.

Reply:
Hi Emmy, I'm thankful for the review making us into better software developers, however  kindly clarify why i cant use the id tag but you want me to use class tag. Pardon me I'm asking this cause i didnt see it on the project requirements.

Reply:
Hi, id is used for single elements that appear on the page for only once, whereas class is used for single or multiple elements that appear on the page for once or more than once, this means that it is better approach to use class instead of id however you can use it while you are planning to include Js later in the project.